addons/website_slides/static/src/xml/website_slide_upload.xml

```diff
@@ -0,0 +1,150 @@
+                    <div class="controls col-md-10">
+                        <input id="tag_ids" name="tag_ids" type="hidden"/>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </t>
+
+    <t t-name="website.slide.upload.modal.webpage">
+        <div>
+            <form class="clearfix">
+                <div class="form-group row">
+                    <label for="name" class="col-form-label col-md-2">Title</label>
+                    <div class="col-md-10">
+                        <input id="name" placeholder="Title" class="form-control" required="required"/>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="category_id" class="col-form-label col-md-2">Category</label>
+                    <div class="controls col-md-10">
+                        <input class="form-control" id="category_id"/>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="tag_ids" class="col-form-label col-md-2">Tags</label>
+                    <div class="controls col-md-10">
+                        <input id="tag_ids" name="tag_ids" type="hidden"/>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </t>
+
+    <t t-name="website.slide.upload.modal.video">
+        <div>
+            <form class="clearfix">
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <div class="img-thumbnail">
+                            <div class="o_slide_preview">
+                                <img src="/website_slides/static/src/img/document.png" id="slide-image" title="Content Preview" alt="Content Preview" class="img-fluid"/>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <ul class="list-group">
+                            <li class="list-group-item">
+                                <h5 class="list-group-item-heading">
+                                    <label for="url" class="col-form-label">Youtube Link</label>
+                                </h5>
+                                <input id="url" name="url" class="form-control" placeholder="Youtube Video URL" required="required"/>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <canvas id="data_canvas" style="display: none;"></canvas>
+                <div class="form-group row">
+                    <label for="name" class="col-form-label col-md-2">Title</label>
+                    <div class="col-md-10">
+                        <input id="name" placeholder="Title" class="form-control" required="required"/>
+                    </div>
+                </div>
+                <div class="form-group row">
```

Comment:
Since every slide_type has "category" and "tags", I wonder if we should put them in another template and use "t-call"?
That would reduce the amount of code and could be reused for other types of slides ("certification" for example).

Reply:
Maybe but I think it's only gonna make inheritance for some cases more complicated. I suggest we keep it like this for now, to see if a new complex slide type appears anytime soon. Otherwise, we will see to factorize some stuff here.

Reply:
+1 with @awa-odoo it is duplicated a lot of times. It will simplify futur update of upload widget as I am quite certain things will change.